app/components/display-message.tsx

```diff
@@ -1,11 +1,12 @@
 import { Center, Flex, Image, Heading } from '@chakra-ui/react';
 
-interface DisplayPageProps {
+interface DisplayMessageProps {
   img: string;
   desc: string;
+  alt: string;
 }
 
-export default function DisplayPage({ img, desc }: DisplayPageProps) {
+export default function DisplayMessage({ img, desc, alt }: DisplayMessageProps) {
```

Comment:
This name is no better.  What you have here is a component for showing an image with a description.  `ImageWithMessage`?